src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/RssLink.java

```diff
@@ -8,7 +8,7 @@
     private final Blog blog;
 
     public RssLink(Blog blog) {
-        Preconditions.checkArgument(blog != null, "Person can not be bull");
+        Preconditions.checkArgument(blog != null, "Person can not be null");
```

Comment:
Why `Person can not be null` and not `blog can not be null`?

You can avoid this line if you use [`@NonNull`](https://projectlombok.org/features/NonNull.html) annotation on a `blog` parameter.